saltysimulations/salty-747
# Bank Angle Selector knob is set to AUTO by default to improve LNAV tracking

Changes:

![BAS](https://user-images.githubusercontent.com/61822722/159269125-45aa906f-93ce-4dc9-b8fc-88863263a505.png)

- Added MaxBankIndex to missing files from the runway.FLT 

Testing Instructions:

1. Start the sim fresh and spawn C&D.
2. Observe that the Bank Angle Selector knob is set to AUTO upon spawning.
